extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getService} from '../../../src/service';
+import {viewerFor} from '../../../src/viewer';
+import {Observable} from '../../../src/observable';
+
+/**
+ * @callback AnalyticsEventListener
+ * @param {AnalyticsEvent} event that got triggered.
+ */
+
+/**
+ * @param {Window} window Window object to listen on.
+ * @param {!Event} type Event type to listen to.
```

Comment:
I was reacting to the global Event type in JS. With the new name, you can certainly use enum type here - that would be better.